pageserver/src/tenant/remote_timeline_client/download.rs

```diff
@@ -226,38 +224,34 @@ pub(super) async fn download_index_part(
     storage: &GenericRemoteStorage,
     tenant_id: &TenantId,
     timeline_id: &TimelineId,
+    generation: Generation,
 ) -> Result<IndexPart, DownloadError> {
-    let index_part_path = conf
+    let local_path = conf
         .metadata_path(tenant_id, timeline_id)
         .with_file_name(IndexPart::FILE_NAME);
```

Comment:
`index_part.json` doesn't exist locally. Let's not cause confusion

Reply:
Fixed in 2e03e9e8b91601a996e2113288f4cbeb311fd1fa (there was an existing bug that the imaginary local path was used in the context() strings, cleaned those up too.)